### PR TITLE
Add properties to config file. Implement error handling

### PR DESCRIPTION
**Pull request by MartynaUb**

### CODE REVIEW

File: Support_Your_Locals/Views/Business/AddAdvertisement.cshtml [hunk 1]
```diff
@@ -93,11 +96,11 @@
         attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
         maxZoom: 18,
         id: 'mapbox/streets-v11',
-       tileSize: 512,
+        tileSize: 512,
         zoomOffset: -1,
-        accessToken: 'pk.eyJ1IjoibW1hcmdlIiwiYSI6ImNrZ3NmOTVrOTAwdmIzMHBlbm54MW8xYXAifQ.4Y4ElvNj24N3eMn28RhKNA'
+        accessToken: '@Configuration["Maps:AccessToken"]'
```
Comment (MartynaUb) [inline Support_Your_Locals/Views/Business/AddAdvertisement.cshtml:116]: This value is extracted from the user secret manager

File: Support_Your_Locals/Controllers/HomeController.cs [hunk 2]
```diff
@@ -12,11 +13,12 @@ public class HomeController : Controller
     {
 
         private IServiceRepository repository;
-        public int PageSize = 4;
+        public int PageSize;
```
Comment (EvalVis) [inline Support_Your_Locals/Controllers/HomeController.cs:16]: This should be more than 0. Otherwise, business will not show.

Comment (EvalVis) [reply Support_Your_Locals/Controllers/HomeController.cs:16]: Oh, just seen its in config.

File: Support_Your_Locals/Controllers/HomeController.cs [hunk 3]
```diff
@@ -41,5 +43,16 @@ public ViewResult Index(SearchResponse searchResponse, string product, int page
             });
         }
 
+        public ViewResult Error(string code)
+        {
+            switch (code)
+            {
+                case "403":
+                    return View("Errors/Forbidden");
+                case "404":
```
Comment (EvalVis) [inline Support_Your_Locals/Controllers/HomeController.cs:54]: Errors for users. That sounds nice.

**Review by Yar3k — APPROVED** [review]
Aproved 👍

**Pull request merged**